kubernetes-aws-yaml/Pulumi.yaml

```
@@ -0,0 +1,52 @@
+name: k8s-aws-yaml
+runtime: yaml
+description: A Pulumi YAML program to stand up a Kubernetes cluster on AWS
+template:
+  config:
+    aws:region:
+      description: The AWS region to deploy into
+      default: us-west-2
+configuration:
+  minClusterSize:
+    type: Number
+    default: 3
+  maxClusterSize:
+    type: Number
+    default: 6
+  desiredClusterSize:
+    type: Number
+    default: 3
```

When I run this, I get complaints about the default not being a string. I _think_ that means these should be:

 ```suggestion
  minClusterSize:
    type: Number
    default: "3"
  maxClusterSize:
    type: Number
    default: "6"
  desiredClusterSize:
    type: Number
    default: "3"
```

---

@cnunciato I can't reproduce this on my system. When I run `pulumi new <path-to-template>` and then run `pulumi up` from the resulting new project, I don't get any messages or complaints about the defaults.